flowdb/sql/04_schema_other.sql

```diff
@@ -68,3 +68,15 @@ Schema used for temp storage during etl.
 */
 
 CREATE SCHEMA IF NOT EXISTS etl;
+
+CREATE TYPE cdrtype AS ENUM ('voice', 'sms', 'mds');
+CREATE TYPE status AS ENUM ('in_process','done', 'quarantine');
+CREATE TABLE etl.etl (
+        id SERIAL NOT NULL,
+        file_name VARCHAR,
+        cdr_type cdrtype,
+	cdr_date DATE,
+        status status,
+        time_stamp TIMESTAMP WITHOUT TIME ZONE,
```

Comment:
Any specific reason to use timestamp **without** time zone?

Reply:
Irrelevant really when recording etl?

Reply:
Sure, so why not use `WITH TIME ZONE` so that timestamps are automatically converted to UTC? Agreed that it's not a big deal, but seems to be good practice to always use timestamps with time zone except in rare cases so we might as well go with the standard?

Reply:
Yeah ok...